collector/src/execute.rs

```diff
@@ -1132,6 +1138,51 @@ impl<'a> Processor for ProfileProcessor<'a> {
                 fs::copy(&tmp_eprintln_file, &eprintln_file)?;
             }
 
+            // mono item results are redirected (via rustc-fake) to a file
+            // called `mono-items`. We copy it from the temp dir to the output
+            // dir, giving it a new name in the process.
+            Profiler::MonoItems => {
+                let tmp_file = filepath(data.cwd.as_ref(), "mono-items");
+                let out_dir = self.output_dir.join(&out_file("mono-items"));
+                let _ = fs::create_dir_all(&out_dir);
+                let result_file = filepath(&out_dir, "raw");
+
+                fs::copy(&tmp_file, &result_file)?;
```

Comment:
Are these result files severed from the site or is this currently only useable locally?

Reply:
These are all just usable locally, the perf collector server doesn't hit this function at all, I believe.

Eventually I'd like to surface this specific profile in the site - potentially integrated with the recently added codegen schedule graphs - but that's future work.